SECURITY.WHITEPAPER.md

```diff
@@ -0,0 +1,113 @@
+# Datasafe Security Whitepaper
+
+## General information
+Datasafe is a flexible encryption library. It uses different encryption algorithms. Some of them can be 
+configured by client application. It uses encryption algorithms provided by Bouncy Castle library by default.
+CMS (crypto message syntax) standard [RFC5652](https://tools.ietf.org/html/rfc5652.html) employed for storing private 
+files encrypted with symmetric keys as well as for sharing files with other users using asymmetric key pairs. 
+
+## Default storage configuration
```

Comment:
Storage is irrelevant to security to whitepaper - remove it, but keep user profile info